mmv1/third_party/terraform/tests/resource_cloudids_endpoint_test.go

```diff
@@ -25,9 +25,19 @@ func TestAccCloudIdsEndpoint_basic(t *testing.T) {
 				Config: testCloudIds_basic(context),
 			},
 			{
-				ResourceName:      "google_cloud_ids_endpoint.endpoint",
-				ImportState:       true,
-				ImportStateVerify: true,
+				ResourceName:            "google_cloud_ids_endpoint.endpoint",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"threat_exceptions"},
```

Comment:
We do not need to ignore `threat_exceptions`?

Reply:
I'm getting some 
    provider_test.go:307: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.
        
        (map[string]string) {
        }
        
        
        (map[string]string) (len=3) {
         (string) (len=19) "threat_exceptions.#": (string) (len=1) "2",
         (string) (len=19) "threat_exceptions.0": (string) (len=2) "12",
         (string) (len=19) "threat_exceptions.1": (string) (len=2) "67"
        }

Reply:
Yeah, you can try after changing the name to `threatExceptions` in api.yam file.